src/MahApps.Metro/Styles/Controls.DatePicker.xaml

```diff
@@ -307,4 +307,173 @@
+        <Setter Property="BorderBrush" Value="{DynamicResource MahApps.Brushes.InputBorder.Mui}" />
+        <Setter Property="BorderThickness" Value="1" />
+        <Setter Property="IsTodayHighlighted" Value="True" />
+        <Setter Property="SelectedDateFormat" Value="Short" />
+        <Setter Property="Padding" Value="2,2,2,1"/>
+        <Setter Property="HorizontalContentAlignment" Value="Stretch" />
+        <Setter Property="Focusable" Value="True" />
+
+        <Setter Property="CalendarStyle">
+            <Setter.Value>
+                <Style TargetType="Calendar" BasedOn="{StaticResource {x:Type Calendar}}">
+                    <Setter Property="Template">
+                        <Setter.Value>
+                            <ControlTemplate TargetType="Calendar">
+                                <Grid Name="PART_Root" Margin="4" HorizontalAlignment="Center">
+                                    <Border Background="#0C000000" BorderBrush="#0F000000" BorderThickness="1" CornerRadius="2.5" Margin="-1" />
+                                    <Border Background="#0A000000" BorderBrush="#0C000000" BorderThickness="1" CornerRadius="2.75" Margin="-2" />
+
+                                    <CalendarItem Name="PART_CalendarItem"
+                                                  Background="{DynamicResource MahApps.Brushes.PopupBackground.Mui}"
+                                                  Style="{TemplateBinding CalendarItemStyle}"
+                                                  BorderThickness="0"
+                                                  Margin="1"/>
+
+                                    <Border BorderBrush="{DynamicResource MahApps.Brushes.Accent.Mui}" BorderThickness="1" />
+                                </Grid>
+                            </ControlTemplate>
+                        </Setter.Value>
+                    </Setter>
+                </Style>
+            </Setter.Value>
+        </Setter>
+
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="{x:Type DatePicker}">
+                    <Border x:Name="LayoutRoot" BorderBrush="{TemplateBinding BorderBrush}"
+                            BorderThickness="{TemplateBinding BorderThickness}"
+                            Background="{TemplateBinding Background}"
+                            Padding="{TemplateBinding Padding}">
+                        <Grid x:Name="PART_Root"
+                              HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
+                              VerticalAlignment="{TemplateBinding VerticalContentAlignment}">
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="*" />
+                                <ColumnDefinition Width="Auto" />
+                            </Grid.ColumnDefinitions>
+                            <!-- TODO: MUI - Replace or implement this control -->
```

Comment:
TODO

Reply:
@punker76 I can either use a metro circle button here or import this `ModernButton` from MUI, any preference?